core/storage/suggestion/gae_models.py

```diff
@@ -3506,6 +3506,26 @@ class TranslationCoordinatorsModel(base_models.BaseModel):
     coordinators_count = datastore_services.IntegerProperty(
         indexed=True, required=True)
 
+    # Here we use MyPy ignore because the signature of this method
+    # doesn't match with BaseModel.get().
```

Comment:
Was there any specific reason for creating a separate model and not using the existing `UserContributionRights` model?

Reply:
`UserContributionRights` have one instance per user and the new model has one instance per language. We are displaying stats of all the languages at once in the new dashboard so it would have been to many calls to the datastore. [Mock Link](https://www.figma.com/file/bmyOkQwosePcoNAr45Rdk0/Admin-Dashboard?node-id=35%3A3752&mode=dev) to get a better idea of what we are planning to do.

`UserContributionRights` stores contribution rights of the user and the new model stores administrative rights of the user